pkg/config/config_template.yaml

```diff
@@ -1483,6 +1483,17 @@ api_key:
   #
   # trace_buffer: 0
 
+  ## @param receiver_timeout - integer - optional - default: 5
+  ## @env DD_APM_RECEIVER_TIMEOUT - integer - optional - default: 5
```

Comment:
If we list a default in the config template we need to add it to the config too ([here](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/setup/apm.go#L101)).